messenger.rst

```diff
@@ -860,7 +866,13 @@ The ``amqp`` transport configuration looks like this:
     # .env
     MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
 
+    # or use the AMQPS protocol
+    MESSENGER_TRANSPORT_DSN=amqps://guest:guest@localhost/%2f/messages
+
+
 To use Symfony's built-in AMQP transport, you need the AMQP PHP extension.
+If you want to use TLS/SSL encrypted AMQP you must provide a CA certificate. You should set it through ``amqp.cacert = /etc/ssl/certs`` (path depends on your system) in your ``php.ini`` file or by setting the ``cacert`` setting (e.g ``amqps://localhost?cacert=/etc/ssl/certs/``)
```

Comment:
```suggestion
If you want to use TLS/SSL encrypted AMQP you must provide a CA certificate. You need to set it using ``amqp.cacert = /etc/ssl/certs`` (path depends on your system) in your ``php.ini`` file or by setting the ``cacert`` parameter (e.g ``amqps://localhost?cacert=/etc/ssl/certs/``)
```

Reply:
Setting is actually used in the same file below. In think we should use the same word for the same thing no ?

Reply:
When dealing with URIs with query strings, the word "setting" is never used for anything after the `?`, and I find it confusing in a Symfony context, as a setting is something to go in a config file.

RFC3986 refers to
> "key=value" pairs
while [Wikipedia](https://en.wikipedia.org/wiki/Query_string) talks of
> multiple parameters in the query string, separated by some delimiter

In the end it comes down to terminology that Symfony chooses to use. My original reason for the edit was having two 'setting's in one sentence isn't considered good English :)